include/uzfs_io.h

```diff
@@ -22,9 +22,13 @@
 #ifndef	_UZFS_IO_H
 #define	_UZFS_IO_H
 
-/* writes metadata 'md' to zil records */
+/*
+ * writes metadata 'md' to zil records
+ * is_rebuild: if io is from replica or for rebuilding then is_rebuild should
```

Comment:
if IO is from target, it will be set to FALSE, else, it should TRUE

Reply:
ok